lib/Service/ExternalStorageService.php

```diff
@@ -62,8 +62,8 @@ public function getMetadataForSpecificFile(IUser $nextcloudUser, int $fileId): a
 
             if (!isset($externalStorageConfiguration['error'])) {
                 $requestBody = [
-                    "user" => $nextcloudUser->getUID(),
-                    "filepath" => $externalStorageConfiguration['filepath'],
+                    "verbose" => true,
+                    "filePath" => $externalStorageConfiguration['filepath'],
```

Comment:
field is case insensitive.
You can keep lowercase here if you prefer.

Reply:
I prefer to leave the capitals like that in camel case, if it doesn't change anything.